database/migrations/20231109103424_courses_table.js

```diff
@@ -0,0 +1,33 @@
+// courses_table.js
+
+const instructors = require('./20231109103300_instructors_table');
+
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('courses', function(table) {
+      table.bigint('id').primary().notNullable();
+      table.string('name', 100).notNullable();
+      table.string('description', 256).notNullable();
+      table.enum('course_level', ['Beginner', 'Advanced']).notNullable();
```

Comment:
Good job, but there is a little change that needs to be made.
## Status: Request change.

Kindly remove line 16 from the course migration file, Based on the recent change from plan to package this won't be needed.

## Highlight:
- Gitflow best practice ✔
- JavaScript best practice ✔
- Good commit history ✔

Happy coding 🎉🎯